core/textile/client.go

```diff
@@ -351,12 +351,12 @@ func (tc *textileClient) IsRunning() bool {
 func (tc *textileClient) healthcheck(ctx context.Context) {
 	log.Debug("Textile Client healthcheck... Start.")
 
+	tc.checkHubConnection(ctx)
```

Comment:
@jsonsivar 

What do you think about documenting the trade-off?

Reply:
Good idea, will add a comment in the code before merging.

Reply:
Do we need to return if we get error here? otherwise if you are offline, that call will fail and then the initialize will still not create the mirror bucket but it will create the local one thus not ever creating the mirror one again

Reply:
Fixed in [9e3dec9](https://github.com/FleekHQ/space-daemon/pull/159/commits/9e3dec9634c08fe5cd7e71f3ba487f353dc63000). 

I set it so that the init only happens is there is no error on the hub connection so if you are offline it will keep trying to init until you are online again.  Not ideal but we can add a better fix after further discussion so we can support offline/online better as per the comment.